styles/components/_showcode.scss

```diff
@@ -0,0 +1,34 @@
+.showCode {
```

Comment:
Relying on class names should be enough. I know it's global CSS but it's not too bad. 🎱